### PR TITLE
Integrates xvectors extraction

### PR DESCRIPTION
**Pull request by aromanusc**
- Integrates xvector feature extraction

### CODE REVIEW

**Comment by adrianSRoman** [comment]
- Validated the implementation by extracting several thousands of audio embeddings from the 2D3MF dataset

File: preprocess/extract_features.py [hunk 1]
```diff
@@ -215,11 +224,6 @@ def extract_audio(audio_path, audio_model, n_feats):
 
                 # Audio Feature Extraction
                 dup = False
-                if args.Forensics:
```
Comment (adrianSRoman) [inline preprocess/extract_features.py:218]: When is this used @aiden200?

Comment (aiden200) [reply preprocess/extract_features.py]: not used anymore

**Review by aiden200 — APPROVED** [review]
lgtm

**Pull request merged**